formula/nicotine-plus.rb

```diff
@@ -0,0 +1,42 @@
+class NicotinePlus < Formula
+  include Language::Python::Virtualenv
+
+  desc "is a graphical client for the SoulSeek peer-to-peer system."
```

Comment:
Please don’t start the description with `is`: `"Graphical client for the SoulSeek peer-to-peer system"`.